frontend/components/product-list/sections/FilterableProductsSection/index.tsx

```diff
@@ -236,9 +248,13 @@ const ProductListEmptyState = forwardRef<EmptyStateProps, 'div'>(
                   Empty collection
                </Text>
                <Text maxW="500px" color="gray.500" textAlign="center" px="2">
-                  This collection does not have products. Try to navigate to
-                  subcategories to find what you are looking for.
+                  This collection does not have products.
                </Text>
+               {parentCategory && (
+                  <Link href={parentCategory.path}>
+                     Return to parent category
```

Comment:
Can we use the title here?
```suggestion
                     Return to {parentCategory.title}
```